okhttp/src/main/java/io/grpc/okhttp/OkHttpClientTransport.java

```diff
@@ -1171,7 +1172,7 @@ public void data(boolean inFinished, int streamId, BufferedSource in, int length
       }
 
       // connection window update
-      connectionUnacknowledgedBytesRead += length;
+      connectionUnacknowledgedBytesRead += paddedLength;
```

Comment:
This is just the connection window. The stream window also needs to be handled correctly. That is done within `transportDataReceived()`.

Reply:
Yea it looks so based on rfc
> The entire DATA frame payload is included in flow control, including the Pad Length and Padding fields if present. I

> only [DATA](https://httpwg.org/specs/rfc7540.html#DATA) frames are subject to flow control